templates/leaderboard/leaderboard.html

```diff
@@ -10,8 +10,27 @@
 
 {% block content %}
     <h2>Top Investors</h2>
-    <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
+
+    <body onload="leaderLoadFunction()" style="margin:0;">
```

Comment:
You can't have a `<body>` tag here since this is already in the middle of the body.